src/pages/ships/components/BuyShips.js

```diff
@@ -0,0 +1,132 @@
+import { useState, useEffect } from 'react'
+import { loadFromLocal } from '../../../helper/localStorage'
+import styled from 'styled-components'
+import axios from 'axios'
+import PropTypes from 'prop-types'
+
+BuyShips.propTypes = {
+  manufacturer: PropTypes.string,
+  maxCargo: PropTypes.number,
+  plating: PropTypes.number,
+  speed: PropTypes.number,
+  weapons: PropTypes.number,
+  type: PropTypes.string,
+  purchaseLocations: PropTypes.arrayOf(
+    PropTypes.shape({
+      location: PropTypes.string,
+      price: PropTypes.number,
+      system: PropTypes.string,
+    })
+  ),
+  setBuyShips: PropTypes.func,
+}
+
+export default function BuyShips() {
+  const [error, setError] = useState('')
+  const [buyShips, setBuyShips] = useState([])
+  const [success, setSuccess] = useState('')
+  const { token } = loadFromLocal('token')
+
+  useEffect(() => {
+    ;(async () => {
+      const result = await axios({
+        method: 'get',
+        url: 'https://api.spacetraders.io/systems/OE/ship-listings',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${token}`,
+        },
+      })
+      setBuyShips(result.data.shipListings)
+    })()
+  }, [])
+
+  function handleBuyShip(type, location) {
+    try {
+      axios({
+        method: 'post',
+        url: 'https://api.spacetraders.io/my/ships',
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+        data: { type: `${type}`, location: `${location}` },
+      })
+      setSuccess(`One ${type} ship added to your fleet!`)
+    } catch (error) {
+      setError(error.message)
+    }
+  }
+
+  return (
+    <div>
+      <h3>Buy New Ships</h3>
+      {buyShips.map(
+        ({
+          manufacturer,
+          maxCargo,
+          plating,
+          speed,
+          weapons,
+          type,
+          purchaseLocations,
+        }) => (
+          <ShipList>
+            <ul>
+              <li>
+                <strong>
+                  {manufacturer} {type}
+                </strong>
```

Comment:
Suggestion:
Dont use <strong> in HTML try to set the font-weight in css. 
```suggestion
                <StyledSpan>
                  {manufacturer} {type}
                </StyledSpan>
```

Reply:
or use the &:first-child selector to select the first li in this case and set the font-weight to bold